cmd/symbols/squirrel/lang_starlark.go

```
@@ -0,0 +1,184 @@
+	defer squirrel.onCall(node, String(node.Type()), lazyNodeStringer(&ret))()
+	switch node.Type() {
+	case "identifier":
+		return squirrel.getDefStarlarkIdentifier(ctx, node)
+	case "string":
+		return squirrel.getDefStarlarkString(ctx, node)
+	default:
+		return nil, nil
+
+	}
+}
+
+func (squirrel *SquirrelService) getDefStarlarkIdentifier(ctx context.Context, node Node) (ret *Node, err error) {
+	symbol := node.Node.Content(node.Contents)
+	root := Node{
+		RepoCommitPath: node.RepoCommitPath,
+		Node:           getRoot(node.Node),
+		Contents:       node.Contents,
+		LangSpec:       node.LangSpec,
+	}
+	exports, err := starlarkBindingNamed(symbol, root)
+	if err != nil {
+		return nil, err
+	}
+	if exports != nil {
+		return &Node{
+			RepoCommitPath: node.RepoCommitPath,
+			Node:           exports,
+			Contents:       node.Contents,
+			LangSpec:       node.LangSpec,
+		}, nil
+	}
+	fmt.Println(node)
+	return nil, nil
+}
+
+func (squirrel *SquirrelService) getDefStarlarkString(ctx context.Context, node Node) (ret *Node, err error) {
+	sitterQuery, err := sitter.NewQuery([]byte(loadQuery), node.LangSpec.language)
+	if err != nil {
+		return nil, errors.Newf("failed to parse query: %s\n%s", err, loadQuery)
+	}
+	defer sitterQuery.Close()
+	cursor := sitter.NewQueryCursor()
+	defer cursor.Close()
+	cursor.Exec(sitterQuery, getRoot(node.Node))
+
+	for {
+		match, ok := cursor.NextMatch()
+		if !ok {
+			return nil, nil
+		}
+
+		if len(match.Captures) < 3 {
+			panic(match.Captures)
+		}
+		path := getStringContents(swapNode(node, match.Captures[1].Node))
+		symbol := getStringContents(swapNode(node, match.Captures[2].Node))
+		if nodeId(match.Captures[2].Node) != nodeId(node.Node) {
+			return nil, nil
+		}
+
+		pathComponents := strings.Split(path, ":")
+
+		if len(pathComponents) != 2 {
+			return nil, nil
+		}
+
+		directory := pathComponents[0]
+		filename := pathComponents[1]
+
+		if !strings.HasPrefix(directory, "//") {
+			panic(directory)
```

You can return an error here, too:

```suggestion
			return nil, errors.Newf("expected starlark directory to be prefixed with \"//\", got %q", directory)
```